tests/integration/test_bamcounts.yml

```diff
@@ -0,0 +1,15 @@
+- name: test_get_bamcounts 
```

Comment:
It looks like by convention we should add this to stampipes.yml as well. Can leave wdl name the same.

Reply:
should be good now, sorry about that.